frontend/src/utils/string.ts

```diff
@@ -0,0 +1,4 @@
+const capitalizeFirstLetter = (s: string) =>
+  s.charAt(0).toUpperCase() + s.slice(1).toUpperCase();
```

Comment:
should this be `s.slice(1).toLowerCase()`?

Reply:
yes it should be, good spot.

We can probably handle Sklodowska-Curie very easily by using the hypens. handling von Bismark requires either never changing name capitalization or having tons of exceptions. So this is one for the PO to weigh in on